### PR TITLE
root: New Version, Better Variant Handling

### PR DESCRIPTION
**Pull request by ChristianTackeGSI**
* Add version 6.20.04
* This version still requires numpy for +python
* ROOT 6.20 renamed the define from `python` to `pyroot`: Add appropiate handling

Maintainer-Ping: @chissg @HadrienG2 @drbenmorgan

### CODE REVIEW

**Review by adamjstewart — APPROVED** [review]
Just waiting for maintainer approval before merging.

**Review by HadrienG2 — APPROVED** [review]
Looks good to me!

**Review by alalazo — APPROVED** [review]

**Pull request merged**
